doc/man3/BIO_read.pod

```diff
@@ -72,19 +90,21 @@ a retry instead of blocking.
 See L<BIO_should_retry(3)> for details of how to
 determine the cause of a retry and other I/O issues.
 
-If the BIO_gets() function is not supported by a BIO then it possible to
-work around this by adding a buffering BIO L<BIO_f_buffer(3)>
-to the chain.
+If the "gets" method is not supported by a BIO then BIO_get_line() can be used.
+In this case it is also possible to use BIO_gets()
```

Comment:
This seems confusing a little. I'd use something like: `It is also possible to make BIO_gets() usable by ...`

Reply:
Yep, done.